doc/api/repl.md

```diff
@@ -162,6 +162,17 @@ Expression assignment to _ now disabled.
 4
 ```
 
+Similarly, `_err` will refer to the last seen error, if there was any.
```

Comment:
Is there any way we could add a Changes section for this?

Reply:
@evanlucas done!